lib/background/fetch.js

```diff
@@ -1,48 +1,23 @@
 import $ from 'jquery';
 
-const loader = (urls, cb) => {
-  const { url, method = 'HEAD' } = urls.shift();
-
-  $.ajax({
-    method,
-    url,
-  }).then((res) => {
-    chrome.runtime.sendMessage({
-      type: 'track',
-      payload: {
-        category: 'fetch',
-        action: 'success',
-      },
-    });
-
-    cb(null, url, res);
-  }).fail(() => {
-    if (urls.length === 0) {
-      chrome.runtime.sendMessage({
-        type: 'track',
-        payload: {
-          category: 'fetch',
-          action: 'error',
-        },
+export default async (urls) => {
```

Comment:
Good point, fixed in https://github.com/OctoLinker/browser-extension/pull/306/commits/b896d9dea3d43eb4cf405337c34ea15104e94feb